Documentation/SpatialAwareness/UsageGuide.md

```diff
@@ -0,0 +1,55 @@
+# Spatial Awareness - Usage Guide
+
+This document contains some guides for common tasks that programmatically access the spatial awareness system.
+
+## Enumerating and accessing the meshes
+
+Accessing the meshes that are currently known to the spatial awareness system involves first querying
+for the spatial awareness system, then getting a hold of the IMixedRealitySpatialAwarenessMeshObserver
+(note that the base spatial awareness observer has no notion of meshes), and then enumerating those
+meshes known that observer.
```

Comment:
```suggestion
meshes known to that observer.
```

Reply:
Thanks, for future reference I don't think we need to put ourselves as waiting for a single grammar fix (of which the solution was provided here).